### DISCUSSION TITLE
Diskrepansi Performa Model Random Forest pada Data Train, Data Validation, dan Data Test pada Masalah Concrete Prediction

### DISCUSSION THREAD

**Discussion by MiftahulLabiib** [teamalgoritma/community#729] [Q&A] [answered]
Untuk mendapatkan performa model yang cukup baik, saya menggunakan randomforest untuk menyelesaikan masalah concrete prediction pada ML capstone project. Pada tahap evaluasi model, data train yang saya gunakan dalam pemodelan memperoleh performa yang cukup baik dimana R2 nya sebesar 98% dan MAE nya sebesar 1,69. Untuk mengecek performa pada unseen data, saya kemudian memprediksi data validation dan menemukan bahwa nilai R2 nya sebesar 93% dan MAE nya 2,9. Oleh karena itu, saya merasa bahwa model ini sudah cukup baik dan memprediksi data test yang akan disubmit ke leaderboard namun ketika saya melakukan submission di leaderboard menunjukkan hasil prediksi data test saya memiliki MAE 23 dan R2 hanya 15%. Apa kemungkinan kesalahan yang terdapat pada model ini?

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @MiftahulLabiib,

Kemungkinan hasil prediksi yang dikirimkan ke submission kurang tepat (data dan hasil prediksi tidak sesuai) melihat nilai-nilai metrics yang sangat rendah. Silahkan dicek terlebih dahulu untuk memastikan kesesuaian submissionnya.
